comp/metadata/inventoryagent/inventoryagentimpl/inventoryagent.go

```diff
@@ -180,6 +181,11 @@ func (ia *inventoryagent) initData() {
 	ia.data["agent_version"] = version.AgentVersion
 	ia.data["agent_startup_time_ms"] = pkgconfigsetup.StartTime.UnixMilli()
 	ia.data["flavor"] = flavor.GetFlavor()
+	if val, err := fips.Enabled(); err == nil {
+		ia.data["fips_mode"] = val
+	} else {
```

Comment:
We our metadata payload should be constant, so even it we get an error the field should be present in the payload.